app/_includes/nav.html

```diff
@@ -1,147 +1,770 @@
+              >
+                <a
+                  target="_blank"
+                  href="https://insomnia.rest/"
+                  class="nav-link"
+                  ><img src="/assets/images/nav/insomnia.svg" /> Insomnia</a
+                >
+              </li>
             </ul>
           </li>
-          <li class="multicol wide nav1 menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-12481 navbar-item docs">
-            <a target="_blank" href="https://docs.konghq.com/?itm_source=website&itm_medium=nav" class="nav-link">Docs</a>
+          <li
+            class="multicol wide nav1 menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-12481 navbar-item docs"
+          >
+            <a
+              target="_blank"
+              href="https://docs.konghq.com/?itm_source=website&itm_medium=nav"
+              class="nav-link"
+              >Docs</a
+            >
             <ul class="sub-menu">
-              <li class="nav2 menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-12482 navbar-item">
+              <li
+                class="nav2 menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-12482 navbar-item"
+              >
                 <a class="nav-link">Get Started</a>
                 <ul class="sub-menu">
-                  <li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-12484 navbar-item"><a href="https://konghq.com/install?itm_source=website&itm_medium=nav" class="nav-link"><img src="/assets/images/nav/install-thick.svg"> Install Kong Gateway</a></li>
-                  <li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-12485 navbar-item"><a target="_blank" href="https://docs.konghq.com/enterprise/1.5.x/deployment/installation/overview/?itm_source=website&itm_medium=nav" class="nav-link"><img src="/assets/images/nav/getting-started-thick.svg"> Install Kong Enterprise</a></li>
+                  <li
+                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-12484 navbar-item"
+                  >
+                    <a
+                      href="https://konghq.com/get-started#install?itm_source=website&itm_medium=nav"
+                      class="nav-link"
+                      ><img src="/assets/images/nav/install-thick.svg" />
+                      Install Kong Gateway</a
+                    >
+                  </li>
+                  <li
+                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-12485 navbar-item"
+                  >
+                    <a
+                      target="_blank"
+                      href="https://docs.konghq.com/enterprise/1.5.x/deployment/installation/overview/?itm_source=website&itm_medium=nav"
```

Comment:
```suggestion
                      href="https://docs.konghq.com/enterprise/latest/deployment/installation/overview/?itm_source=website&itm_medium=nav"
```
Can we change this to `latest` both here and on konghq.com? That way the link won't need constant updating (on konghq.com, it currently points to 1.3.x).

Reply:
Got it!